packages/optimizer/lib/transformers/RewriteAmpUrls.js

```diff
@@ -56,7 +56,7 @@ const {calculateHost} = require('../RuntimeHostHelper');
  */
 class RewriteAmpUrls {
   constructor(config) {
-    this.esmModulesEnabled = config.experimentEsm;
```

Comment:
Since you aren't supporting the old version at all, this be a breaking change as far as semver is concerned, right?

Reply:
Good question. My assumption is: the previous default was disabled, the new default is enabled. 

Cases:

* If someone previously enabled it, it's still going to be enabled => not a breaking change
* If someone previously explicitly disabled it, it's going to be enabled now. This is in theory a breaking change, however, this is the new "official" default for AMP as it's valid AMP now. And the flag was named experimental + undocumented, so I wouldn't consider it part of the official API.